analytics/src/analytics/integrations/etldb/main.py

```diff
@@ -15,37 +15,47 @@
 from analytics.integrations.etldb.sprint_model import EtlSprintModel
 
 VERBOSE = False
+MIN_SCHEMA_VERSION = 2
 
 
-def init_db() -> None:
+def initialize_database() -> None:
     """Initialize etl database."""
-    # define the path to the sql file
-    parent_path = Path(__file__).resolve().parent
-    sql_path = f"{parent_path}/create_etl_db.sql"
-
-    # read sql file
-    with open(sql_path) as f:
-        sql = f.read()
-
-    # execute sql
+    # get connection to database
     try:
         db = EtlDb()
         cursor = db.connection()
-        cursor.execute(
-            text(sql),
-        )
-        db.commit(cursor)
     except (
         RuntimeError,
         ProgrammingError,
         OperationalError,
         InsufficientPrivilege,
     ) as e:
-        message = f"FATAL: Failed to initialize db: {e}"
+        message = f"FATAL: Failed to initialize database client: {e}"
+        raise RuntimeError(message) from e
+
+    # execute sql
+    try:
+        for sql_path in get_sql_files():
```

Comment:
1. Get the schema version here, as a special case
2. Pass in the schema version as an input into this function
3. Start iterating based on the position in the schema version

Reply:
done